src/customizations/volto/components/manage/Blocks/Image/style.less

```diff
@@ -1,12 +1,126 @@
 .image-block {
   position: relative;
+  width: unset !important;
+
+  .edit {
+    min-height: 70px;
+  }
+
+  .large {
+    min-height: 500px;
+  }
+
+  .medium {
+    min-height: 250px;
+  }
+
+  .small {
+    min-height: 100px;
+  }
+}
+
+.block.align.right .imageElement {
+  margin-right: 0;
+  margin-left: 1em;
+  float: right;
+}
+
+.block.align {
+  .medium {
+    width: 50%;
+  }
+
+  .small {
+    width: 25%;
+  }
+}
+
+.block.align.right .large {
+  width: 50%;
+}
+
+.block.align.right .medium {
+  width: 25%;
+}
+
+.block.align.right .small {
+  width: 10%;
+}
+
+.block.align.left .imageElement {
+  margin-right: 1em;
+  margin-left: 0;
+  float: left;
+}
+
+.block.align.left .large {
```

Comment:
@andreiggr why do you need image css? shouldn't there be already code that sizes the images
when using the align classes?

Reply:
they're used for floating the div with the image. That code I think it targets img elements and that changed to enable the copyright correct positioning.